### ISSUE TITLE
Handling non-json messages without failing

### ISSUE THREAD

**Issue by danielferber** [aldis-ameriks/pino-postgres#1]
Considering an application that uses a postgres table with json/jsonb column to store logs.

The application may print some non json log message. For example, some module that still uses console.log.
In such situation, pino-postgres will fail to commit all messages from the buffer, as postgres will reject the message
This means that all messages within the insert batch are lost.

An alternative would be using a string column on postgres table, but i won't be able anymore to index and search efficiently on json fields.

My suggestion is and option to convert non json message to json, assuring that the json/jsonb field will always receive valid data.

Suggested change: _transform method:
    if (! content.startsWith('{')) {
      const fixedcontent = `{"time":${Date.now()},"msg":"${content}"}`
      buffer.push({ [column]: fixedcontent })
    } else {
      buffer.push({ [column]: content })
    }

**Comment by aldis-ameriks** [comment]
Thanks for the suggestion! Would you like to open a PR for that?

**Comment by danielferber** [comment]
Hi! I created a pull request for the suggestion.
I created an option to enable and disable the proposed feature - disabled by default.
I was unsure what would be best strategy to decide if a string is a valid or invalid JSON, without parsing the entire string.
As I noticed that the parsing was recently removed, I would not suggest adding the parsing back again.
The PR assumes that every potentially string valid JSON string will start with `{"` and end with `}`.
The PR reads the `wrapNonJson` for each messages. I think there could be more efficient implementations. For example, a implementation could pass the line handling as a function/strategy previously created according to the command line parameters.
